app/Http/Controllers/Admin/CategoryController.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use Illuminate\Http\Request;
+use App\Http\Controllers\Controller;
+use App\Models\Category;
+use Session;
+use App\Http\Requests\CategoryRequest;
+
+class CategoryController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $data = Category::orderBy('id', 'DESC')->paginate(config('constant.LIMIT_PAGE_ADMIN'));
+        return view('admin.categories.list', compact('data', $data));
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        return view('admin.categories.add');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param \Illuminate\Http\Request $request of category
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function store(CategoryRequest $request)
+    {
+        $data = new Category;
+        $data ->name = $request ->txtTitle;
```

Comment:
pls name the form request param same to the column in DB. Then no need to assign request to Eloquent. Laravel do the rest
```php
$category = new Category($request->all());
$category->save();
```